tests/lost_reset/test

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/perl
+
+use strict;
+use File::Temp qw/ tempdir tempfile /;
+use Test;
+BEGIN { plan tests => 6 }
+
+###
+# functions
+
+###
+# setup
+
+# reset audit rules
+system("auditctl -D >& /dev/null");
+
+# create stdout/stderr sinks
+(my $fh_out, my $stdout) = tempfile(TEMPLATE => '/tmp/audit-testsuite-out-XXXX',
+				    UNLINK => 1);
+(my $fh_err, my $stderr) = tempfile(TEMPLATE => '/tmp/audit-testsuite-err-XXXX',
+				    UNLINK => 1);
+(my $fh_tmp, my $tmpout) = tempfile(TEMPLATE => '/tmp/audit-testsuite-tmp-XXXX',
+                                    UNLINK => 1);
+
+# Kill the daemon and set the buffers low
+system("kill \$(pgrep -x auditd) 2>/dev/null");
+system("auditctl -b 64 >/dev/null 2>&1");
+
+###
+# tests
+my $result;
+# Start floodping to generate activity
+system("ping -f 127.0.0.1 >/dev/null 2>&1 & echo \$! >$tmpout");
+my $ping_pid = <$fh_tmp>;
+chomp($ping_pid);
+# Add rule to generate audit queue events from floodping
+$result = system("auditctl -a exit,always -S all -F pid=$ping_pid >/dev/null");
+ok($result, 0); # Was the rule added successfully?
+sleep 1;
+kill 'TERM', $ping_pid;
+system("auditctl -d exit,always -S all -F pid=$ping_pid >/dev/null");
+
+# Restart the daemon to collect messages in the log
+system("auditd -n &");
```

Comment:
systemctl doesn't exist on rhel6, but now that I think of it, this feature isn't supported in rhel6.
Instead, we could use "service audit start" since that redirects to systemd.